doc/mgr/ssh.rst

```diff
@@ -0,0 +1,32 @@
+================
+SSH orchestrator
+================
+
+The SSH orchestrator is an orchestrator module that does not rely on a separate
+system such as Rook or Ansible, but rather manages nodes in a cluster by
+establishing an SSH connection and issuing explicit management commands.
+
+Orchestrator modules only provide services to other modules, which in turn
+provide user interfaces.  To try out the SSH module, you might like
+to use the :ref:`Orchestrator CLI <orchestrator-cli-module>` module.
+
+Requirements
+------------
+
+- The Python `remoto` library version 0.35 or newer
+
+Configuration
+-------------
+
+The SSH orchestrator can be configured to use an SSH configuration file. This is
+useful for specifying private keys and other SSH connection options.
+
+::
+
+    # ceph config set mgr mgr/ssh/ssh_config /path/to/config
```

Comment:
Hm, we're saving a path to a file on a specific host. As soon as the active mgr changes, we may no longer to access this file. Would it make sense to store the ssh config itself as a blob?

Reply:
right now i'm following the pattern used to install certificates that's used in the dashboard. but it does make sense to install a specific blob.

Reply:
I added a new option that allows an ssh config file to be imported from an existing file (like `-i file`) and it takes precedence over ones specified as a file system path.